src/version.py

```diff
@@ -0,0 +1,37 @@
+"""
+Version management for pgconsul.
+"""
+
+STATE_FILE = '/tmp/pgconsul.state'
```

Comment:
Why do we need to introduce another file, if we already have status file: https://github.com/yandex/pgconsul/blob/main/src/helpers.py#L242?
Let's just add a field to its json contents.